tar/tarsnap.1-mdoc.in

```diff
@@ -354,6 +354,9 @@ name of the option.
 Do not overwrite existing files.
 In particular, if a file appears more than once in an archive,
 later copies will not overwrite earlier copies.
+.It Fl -keep-going
+(d and print-stats mode only)
+Makes tarsnap continue if one of the archives specified does not exist.
```

Comment:
Grammatical consistency -- the description of an option should be the meaning we're trying to convey to tarsnap.  I'd write this as something like "Continue deleting or printing statistics after finding that one of the archives specified does not exist."